admin-dev/themes/default/template/controllers/search/helpers/view/view.tpl

```diff
@@ -57,8 +57,7 @@ $(function() {
             {foreach $features key=key item=feature}
                 {foreach $feature key=k item=val name=feature_list}
                     <tr>
-                        <td><a href="{$val.link}"{if $smarty.foreach.feature_list.first}><strong>{$key}</strong>{/if}</a></td>
-                        <td><a href="{$val.link}">{$val.value}</a></td>
+                        <td><a href="{$val.link}">{if $smarty.foreach.feature_list.first}><strong>{$key}</strong>{/if}</a></td>
```

Comment:
```suggestion
                        <td><a href="{$val.link}">{if $smarty.foreach.feature_list.first}<strong>{$key}</strong>{/if}</a></td>
```
Missing a `Greater Than Character`

Reply:
I decided to remove the `if` which was actually preventing more than one result item to be displayed at a time.